### PR TITLE
Fix openshift deploy script; readlink doesn't work on osx; and some m…

### PR DESCRIPTION
**Pull request by dejanb**
…ore minor fixes

### CODE REVIEW

File: example/openshift/openshift_deploy.sh [hunk 1]
```diff
@@ -1,13 +1,14 @@
 #!/bin/sh
 
-# Absolute path to this script, e.g. /home/user/bin/foo.sh
-SCRIPT=$(readlink -f "$0")
-# Absolute path this script is in, thus /home/user/bin
-SCRIPTPATH=$(dirname "$SCRIPT")
+
+# Absolute path this script is in
+SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
```
Comment (sophokles73) [inline example/openshift/openshift_deploy.sh]: you/we should also fix this in the `kuberntes_deploy.sh` script then ...

Comment (sophokles73) [inline example/openshift/openshift_deploy.sh]: no  apple fan boys here ;-)

Comment (dejanb) [inline example/openshift/openshift_deploy.sh]: I missed kubernetes deploy script. It's fixed now

**Comment by sophokles73** [comment]
LGTM

**Comment by sysexcontrol** [comment]
Just tested it on Openshift Origin 1.5.0/Ubuntu 16.04.
All looks fine!

**Comment by sysexcontrol** [comment]
Ok, one more thing :-):

The following line breaks the creation of the persistent volume : 
`oc login -u system -p admin -n hono`
IMHO it should be changed back to
`oc login -u system:admin`

At least then everything works as expected again : the persistent volume can be created, `oc get` can list namespaces, etc.

WDYT?

**Comment by dejanb** [comment]
@sysexcontrol I fixed this now. Can you please retest?

**Comment by sysexcontrol** [comment]
@dejanb : tested it successfully: all looks good now!

**Comment by sophokles73** [comment]
@dejanb can you squash and rebase and commit?

**Pull request merged**